### ISSUE TITLE
Kind of scary permissions screen

### ISSUE THREAD

**Issue by weex** [OpenHospitalityNetwork/ohn-solid#8]
When running locally with `yarn start`, I saw this screen which is a lot of text to read and would probably lose a lot of users. In particular the first checkbox with "Read all documents in the Pod" sounds like a lot. Even as someone who I'm pretty sure has no documents on the pod, I get nervous when confronted with such broad statements.

Potential solution(s):
Is there a way to request fewer permissions or to display a simpler view of the situation. 
<img width="1172" alt="Screen Shot 2021-12-14 at 10 32 36 AM" src="https://user-images.githubusercontent.com/38776/146058940-5b6dba4d-d94a-4d75-a13b-f25ef73a39ef.png">
.

**Comment by mrkvon** [comment]
@weex Thank you for reporting!

AFAIK we don't have control over the permission screen you've seen, unfortunately. It's served directly by the Solid Pod Server when you use the app for the first time.

This looks like the screen which is served by Node Solid Server (https://solidcommunity.net, was it?) which has rather terrible UX in general. 😆

If there is a way to limit the items on the screen or change selected defaults, i don't know it.


> "Read all documents in the Pod"

That sounds very broad indeed. There is [Solid Interoperability Panel](https://github.com/solid/data-interoperability-panel) working on [a new specification](https://solid.github.io/data-interoperability-panel/specification/). On first reading it sounded complicated to me and i haven't studied it in detail, but it should allow for finer permission settings.

So hopefully the future is brighter; and once again we can blame the issue on the immaturity of the Solid technology. 😝

**Comment by weex** [comment]
Yes @mrkvon, solidcommunity.net I believe. Maybe if we want to run a pod later we can fix that screen... Or create an issue upstream if there isn't one. One thing I like about text is that it's trivial to update most of the time. But yeah some of these issues are more making notes for the curious vs. expecting a fix.

**Comment by mrkvon** [comment]
We could make this easier for the first-time users, by telling them what to expect before they log in for the first time.

Otherwise we can't do much, since this is a UX issue of https://github.com/nodeSolidServer/node-solid-server, which is generally - to put it mildly - UI-challenged 🙂
